app/uploaders/thumb_image_uploader.rb

```diff
@@ -0,0 +1,51 @@
+# encoding: utf-8
+
+class ThumbImageUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
```

Comment:
I would recommend to type out all code and not use generators, because your brain will remember it better if you use your hands/fingers to work on it.